x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/es_stats/es_stats.tsx

```diff
@@ -1,118 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0; you may not use this file except in compliance with the Elastic License
- * 2.0.
- */
-
-import React, { FunctionComponent, useEffect, useMemo } from 'react';
-import { useHistory } from 'react-router-dom';
-
-import { EuiStat, EuiSpacer, EuiFlexGroup, EuiFlexItem, EuiCard } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
-
-import { reactRouterNavigate } from '../../../../../../../../../src/plugins/kibana_react/public';
-import { getDeprecationsUpperLimit } from '../../../../lib/utils';
-import { useAppContext } from '../../../../app_context';
-import { EsStatsErrors } from './es_stats_error';
-import { NoDeprecations } from '../no_deprecations';
-
-const i18nTexts = {
-  statsTitle: i18n.translate('xpack.upgradeAssistant.esDeprecationStats.statsTitle', {
-    defaultMessage: 'Elasticsearch',
-  }),
-  warningDeprecationsTitle: i18n.translate(
-    'xpack.upgradeAssistant.esDeprecationStats.warningDeprecationsTitle',
-    {
-      defaultMessage: 'Warning',
-    }
-  ),
-  criticalDeprecationsTitle: i18n.translate(
-    'xpack.upgradeAssistant.esDeprecationStats.criticalDeprecationsTitle',
-    {
-      defaultMessage: 'Critical',
-    }
-  ),
-};
-
-interface Props {
-  setIsFixed: (isFixed: boolean) => void;
-}
-
-export const ESDeprecationStats: FunctionComponent<Props> = ({ setIsFixed }) => {
-  const history = useHistory();
-  const {
-    services: { api },
-  } = useAppContext();
-
-  const { data: esDeprecations, isLoading, error } = api.useLoadEsDeprecations();
-
-  const warningDeprecations = useMemo(
```

Comment:
@sabarasaba Do you know why we used `useMemo` here? Was it needed to prevent unnecessary renders, or was it an optimization to prevent recalculating the value unnecessarily?

Reply:
Can't exactly recall why we added it, but its not computationally expensive so we can get rid of it 🔥